src/sidebar/store/use-store.js

```diff
@@ -78,11 +78,13 @@ export function useStoreProxy() {
   const proxy = useRef(/** @type {SidebarStore|null} */ (null));
   if (!proxy.current) {
     // Cached method wrappers.
-    const wrappedMethods = {};
+    /** @type {Map<string, Function>} */
+    const wrappedMethods = new Map();
 
     /**
-     * @param {typeof store} store
-     * @param {string} prop
+     * @template {Store} StoreType
+     * @param {StoreType} store
+     * @param {keyof StoreType & string} prop
```

Comment:
Can you expand on why `& string` is needed here?

Reply:
This filters out the non-string (eg. symbol) keys of the store, which aren't supported by some of the internal logic.